R/SSplotJABBAres.R

```diff
@@ -1,72 +1,41 @@
-#' @param plot Deprecated. Plots (and subplots) are drawn to the active plot device
-#' by default (TRUE), and the option to disable this, via FALSE, is unused.
-#' @param print Deprecated. Please use 'print_plot'.
-#' @param print_plot Option to print to PNG files
-#' @param png Deprecated. please use 'use_png'.
-#' @param use_png Draw plots in PNG format
-#' @param pdf Deprecated. Please use 'use_pdf'.
-#' @param use_pdf option for pdf plots (not tested for TRUE)
-#' @param indexselect Vector of fleet numbers for each model for which to compare
-#' @param miny  minimum abs values of ylim
-#' @param col Optional vector of colors to be used for lines. Input NULL
-#' @param pch Optional vector of plot character values
-#' @param lty Optional vector of line types
-#' @param lwd Optional vector of line widths
-#' @param tickEndYr TRUE/FALSE switch to turn on/off extra axis mark at final
-#' year in time series plots.
-#' @param ylimAdj Multiplier for ylim parameter. Allows additional white space
-#' @param xaxs Choice of x-axis parameter (see ?par for more info)
-#' @param yaxs Choice of y-axis parameter (see ?par for more info)
-#' @param type Type parameter passed to points (default 'o' overplots points on
-#' top of lines)
-#' @param legend Option to add a legend. TRUE by default.
-#' @param legendlabels Optional vector of labels to include in legend.
-#' @param legendloc Location of legend. Either a string like "topleft" or a vector
-#' of two numeric values representing the fraction of the maximum in the x and y
-#' dimensions, respectively. See ?legend for more info on the string options.
-#' @param legendorder Optional vector of model numbers that can be used to have
-#' the legend display the model names in an order that is different than that
-#' which is represented in the summary input object.
-#' @param legendncol Number of columns for the legend.
-#' @param legendcex Allows to adjust legend cex. Defaults to 1.
-#' @param legendsp Space between legend labels
-#' @param legendindex Allows to add legend for selected indices (plots)
-#' @param pwidth Width of plot
-#' @param pheight Height of plot
-#' @param punits Units for PNG file
-#' @param res Resolution for PNG file
-#' @param ptsize Point size for PNG file
-#' @param cex.main Character expansion for plot titles
-#' @param plotdir Directory where PNG or PDF files will be written. By default
-#' it will be the directory where the model was run.
-#' @param filenameprefix Additional text to append to PNG or PDF file names.
-#' It will be separated from default name by an underscore.
-#' @param par list of graphics parameter values passed to par() function
-#' @param verbose Report progress to R GUI?
+#' @param ss3rep Stock Synthesis output as read by [r4SS::SS_output()]
+#' @param subplots string of type of data to plot:
+#' \itemize{
+#'  \item `"cpue"` Index of abundance data
+#'  \item `"len"` Length-composition
```

Comment:
nit pick but add data after length-composition, age-composition, and generalized size-composition